src/components/v2/V2Project/V2PayButton/V2ConfirmPayModal/index.tsx

```diff
@@ -103,6 +103,9 @@ export default function V2ConfirmPayModal({
     if (!weiAmount) return
     await form.validateFields()
 
+    const beneficiaryFormVal = form.getFieldValue('beneficiary')
+    const txBeneficiary = beneficiaryFormVal ? beneficiaryFormVal : beneficiary
```

Comment:
**question** do we need to rely on the `beneficiary` state variable anymore?

I'm wondering if we can just remove it and default to `userAddress` here instead:

```suggestion
    const txBeneficiary = beneficiaryFormVal ? beneficiaryFormVal : userAddress
```

Reply:
Updated